docs/_partials/_install-promise.md

```diff
@@ -188,13 +176,51 @@ To recap the steps you took:
 
 This is only the beginning of working with Promises. Next you will deploy three different Promises to provide a complete solution for an application team.
 
-## Tearing it all down {#teardown}
-To clean up your environment, run the following command:
+## Cleanup environment {#cleanup}
+To clean up your environment you need to delete the Jenkins Resource Requests and the Jenkins Promise.
```

Comment:
This is all correct, but does take two commands. Given the current implementation is to delete any open RR when you delete a Promise, I wonder if we should take advantage to reduce the load on this section?

Reply:
addressed this in my bigger response below 😄